bark_core/signatures/util.py

```diff
@@ -1,18 +1,53 @@
 from gitbark.git import Commit
+from gitbark.cli.util import get_root, click_prompt
+from gitbark.util import cmd
 
 from pgpy import PGPKey, PGPSignature
 from paramiko import PKey
-from typing import Any, Union
+from typing import Any, Union, Optional
+from pygit2 import Repository
 
+import subprocess
+import re
 import warnings
+import os
+import click
 
 warnings.filterwarnings("ignore")
 
 
 class Pubkey:
     def __init__(self, pubkey: str) -> None:
+        self.bytes = pubkey
```

Comment:
Type of pubkey is `str`, but this property is named `bytes`, seems wrong

Reply:
I agree, `pubkey` should be of type `bytes`! Pushed a commit that resolves this.